### PR TITLE
[2.13.x] Made Failsafe Asynchronous

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
 Made Failsafe Asynchronous which fixes downstream itest failures
Master PR https://github.com/codice/ddf/pull/3566
#### Who is reviewing it? 

@GabrielFabian @Bdthomson 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Ask 2 committers to review/merge the PR and tag them here.

@pklinef
@millerw8 
#### How should this be tested?

Build / Unzip / make etc forms directory / Install / turn on experimental features / verify search form crud
#### What are the relevant tickets?
[DDF-4036](https://codice.atlassian.net/browse/DDF-4036)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsLoader.java [hunk 1]
```diff
@@ -100,33 +103,43 @@ public static boolean enabled() {
...
-                    .filter(AttributeGroupMetacard::isAttributeGroupMetacard)
-                    .filter(metacard -> !resultTitles.contains(metacard.getTitle())))
-            .collect(Collectors.toList());
-
-    if (!dedupedTemplateMetacards.isEmpty()) {
-      executeAsSystem(() -> framework.create(new CreateRequestImpl(dedupedTemplateMetacards)));
+      CatalogFramework catalogFramework,
+      EndpointUtil endpointUtil,
+      List<Metacard> systemTemplates) {
+    Subject systemSubject = SECURITY.runAsAdmin(SECURITY::getSystemSubject);
```
Comment (ryeats) [inline catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsLoader.java]: will the credentials need to be refreshed?

**Comment by bdeining** [comment]
build now

**Comment by bdeining** [comment]
build now

**Comment by bdeining** [comment]
build now

File: catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsLoader.java (hunk 1 above)
Comment (bdeining) [reply catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsLoader.java]: I doubt it; this loading only occurs at installation once and never again

**Comment by bdeining** [comment]
already hero'd and fleshed out in 2.14 / master

**Pull request merged**